components/prism-typoscript.js

```diff
@@ -0,0 +1,31 @@
+Prism.languages.typoscript = Prism.languages.extend('javascript', {
```

Comment:
I forget this: Why do you extend JS? TypoScript and JS don't seem to have much in common.

Reply:
Good question. I think this was my starting point i just dont question myself if this has a reason anymore. I remove the dependeny completly. This will be added with my update too.